### PR TITLE
add recipe for prolix-maintenance-bundle

### PR DESCRIPTION
**Pull request by fnordo**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: prolix/maintenance-bundle/2.1/config/packages/lexik_maintenance.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+lexik_maintenance:
+    driver:
+        class: '\Lexik\Bundle\MaintenanceBundle\Drivers\FileDriver'
+        options: {file_path: '%kernel.project_dir%/../var/cache/lock'}
```
Comment (OskarStark) [inline prolix/maintenance-bundle/2.1/config/packages/lexik_maintenance.yaml:4]:
```suggestion
        options:
            file_path: '%kernel.project_dir%/../var/cache/lock'
```

Comment (OskarStark) [reply prolix/maintenance-bundle/2.1/config/packages/lexik_maintenance.yaml:4]: What about using: '%kernel.cache_dir%/lock'

File: prolix/maintenance-bundle/2.1/config/packages/lexik_maintenance.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+lexik_maintenance:
+    driver:
+        class: '\Lexik\Bundle\MaintenanceBundle\Drivers\FileDriver'
```
Comment (OskarStark) [inline prolix/maintenance-bundle/2.1/config/packages/lexik_maintenance.yaml:3]:
```suggestion
        class: 'Lexik\Bundle\MaintenanceBundle\Drivers\FileDriver'
```

File: prolix/maintenance-bundle/2.1/config/packages/lexik_maintenance.yaml (hunk 1 above)
Comment (fnordo) [reply prolix/maintenance-bundle/2.1/config/packages/lexik_maintenance.yaml]: > What about using: '%kernel.cache_dir%/lock' ?

yes, you´re totally right. thx for the suggestion...

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**